tools/plugins/install_go_tools.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+
+. /vagrant/conf.sh
+
+set -e
+
+GOPATH=/home/$VAGRANT_USER/go
+GOBIN=$GOPATH/bin
+
+export GOPATH
+echo "export GOPATH=$GOPATH:/project/sawtooth-core/sdk/go" >> /home/$VAGRANT_USER/.bashrc
+echo "export PATH=$PATH:$GOBIN" >> /home/$VAGRANT_USER/.bashrc
+
+apt-get install -y -q \
```

Comment:
This should be installed in the package_groups/ directory instead of a manual apt-get.